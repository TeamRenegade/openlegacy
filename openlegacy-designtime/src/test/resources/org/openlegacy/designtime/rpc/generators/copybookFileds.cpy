           03 PI1_ESTASK              PIC 9(4).
           03 PI1_ESPEUL              PIC 9(2).
           03 PI1_ESSVIV              PIC 9(2).
           03 PI1_ESKDMK              PIC 9(6).
           03 PI1_ESUSER              PIC X(10).
           03 PI1_ESBOXN              PIC 9(4).
           03 PI1_ESKODB              PIC 9(2).
           03 PI1_ESUSRK              PIC 9(2).
           03 PI1_ESCOMP              PIC 9.
           03 PI1_ESMHOZ              PIC 9(2).
           03 PI1_ESACC1              PIC 9(6).
           03 PI1_ESACC2              PIC 9(6).
           03 PI1_ESFILR              PIC X(50).