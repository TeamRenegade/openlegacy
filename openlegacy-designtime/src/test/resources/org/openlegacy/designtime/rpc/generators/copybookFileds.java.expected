package test.com;

import java.math.BigInteger;
import java.util.List;

import org.openlegacy.annotations.rpc.*;
import org.openlegacy.FieldType.*;
import org.openlegacy.rpc.RpcActions.*;

@RpcPart(name="CopybookFileds")
public class CopybookFileds {
		@RpcNumericField(minimumValue=-9999, maximumValue=9999, decimalPlaces=0)
		@RpcField(length = 4, originalName = "PI1_ESTASK")
		private Integer pi1Estask;

		@RpcNumericField(minimumValue=-99, maximumValue=99, decimalPlaces=0)
		@RpcField(length = 2, originalName = "PI1_ESPEUL")
		private Integer pi1Espeul;

		@RpcNumericField(minimumValue=-99, maximumValue=99, decimalPlaces=0)
		@RpcField(length = 2, originalName = "PI1_ESSVIV")
		private Integer pi1Essviv;

		@RpcNumericField(minimumValue=-999999, maximumValue=999999, decimalPlaces=0)
		@RpcField(length = 6, originalName = "PI1_ESKDMK")
		private Integer pi1Eskdmk;

		@RpcField(length = 10, originalName = "PI1_ESUSER")
		private String pi1Esuser;

		@RpcNumericField(minimumValue=-9999, maximumValue=9999, decimalPlaces=0)
		@RpcField(length = 4, originalName = "PI1_ESBOXN")
		private Integer pi1Esboxn;

		@RpcNumericField(minimumValue=-99, maximumValue=99, decimalPlaces=0)
		@RpcField(length = 2, originalName = "PI1_ESKODB")
		private Integer pi1Eskodb;

		@RpcNumericField(minimumValue=-99, maximumValue=99, decimalPlaces=0)
		@RpcField(length = 2, originalName = "PI1_ESUSRK")
		private Integer pi1Esusrk;

		@RpcNumericField(minimumValue=-9, maximumValue=9, decimalPlaces=0)
		@RpcField(length = 1, originalName = "PI1_ESCOMP")
		private Integer pi1Escomp;

		@RpcNumericField(minimumValue=-99, maximumValue=99, decimalPlaces=0)
		@RpcField(length = 2, originalName = "PI1_ESMHOZ")
		private Integer pi1Esmhoz;

		@RpcNumericField(minimumValue=-999999, maximumValue=999999, decimalPlaces=0)
		@RpcField(length = 6, originalName = "PI1_ESACC1")
		private Integer pi1Esacc1;

		@RpcNumericField(minimumValue=-999999, maximumValue=999999, decimalPlaces=0)
		@RpcField(length = 6, originalName = "PI1_ESACC2")
		private Integer pi1Esacc2;

		@RpcField(length = 50, originalName = "PI1_ESFILR")
		private String pi1Esfilr;

}